### ISSUE TITLE
timedatectl needs a way to print just the timezone

### ISSUE THREAD

**Issue by mpontillo** [systemd/systemd#2033]
If `timedatectl` is to be the official way to get the timezone, a variation of the command to get _just_ the timezone is needed. For example, I would like to do something like:

```
$ timedatectl show-timezone
America/New_York
```

... the requirement being that I would get just the timezone string (as it would be found in /usr/share/zoneinfo), without any extra information.

This way I don't need an atrocious (and possibly fragile) `awk` incantation if I want to use `timedatectl` in favor of reading `/etc/timezone`, checking if `/etc/localtime` is a link, or whatever the system-specifc way to do it is.

Basically, please make it work like `uname -r` or `lsb_release -r -s`.

**Comment by poettering** [comment]
I am not sure timedatectl is really supposed to be a scripting helper. Consider using something like this instead:

busctl get-property org.freedesktop.timedate1 /org/freedesktop/timedate1 org.freedesktop.timedate1 Timezone | cut -d\" -f2
